rmw_dds_common/msg/ParticipantEntitiesInfo.idl

```diff
@@ -4,7 +4,8 @@
 module rmw_dds_common {
   module msg {
     struct ParticipantEntitiesInfo {
-        @key
+        // We're not using keyed topics right now. Commented because is making generation fail when OpenSplice is available.
```

Comment:
@ivanpauno nit: 
```suggestion
        // Commented out because it is making message generation fail when OpenSplice is available.
```

on it's own, separate line.

Reply:
Done in https://github.com/ros2/rmw_dds_common/pull/4/commits/7579823e226e55aaac812b2fa9182dbe1fec5472.